codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1249,6 +1154,27 @@ void removeFromActiveStreams(DefaultStream stream, Iterator<?> itr) {
         boolean allowModifications() {
             return pendingIterations == 0;
```

Comment:
Yep better update comment and use 'reentrance'

> Am 06.05.2016 um 09:41 schrieb Scott Mitchell <notifications@github.com>:
> 
> In codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java:
> 
> > @@ -1249,6 +1154,27 @@ void removeFromActiveStreams(DefaultStream stream, Iterator<?> itr) {
> >          boolean allowModifications() {
> >              return pendingIterations == 0;
> Perhaps my usage of the term "concurrent modification" is misleading. The codec-http2 is not thread safe and what we are really protecting from is a re-entry condition on the same thread which may modify the collection we are iterating. This can happen for example because we call out to user code while iterating.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>